package.json

```diff
@@ -40,9 +40,12 @@
     "nyc": "^14.1.1"
   },
   "dependencies": {
+    "bcrypt": "^3.0.6",
+    "body-parser": "^1.19.0",
```

Comment:
Our app is already using `app.use(express.urlEncoded({ extended: true });`, so need for `body-parser`